sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/autorest.md

```diff
@@ -56,3 +56,143 @@ directive:
+      where: $["definitions"]["ConversationSummarizationTaskParameters"]["properties"]["summaryAspects"]["items"]
+      transform: >
+        $["enum"] = [
+              "issue",
+              "resolution"
+            ];
+```
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["AnalyzeConversationResultsKind"]
+      transform: >
+        $["enum"] = ["conversationalSummarizationResults", "conversationalPIIResults"];
+```
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["AnalyzeConversationSummarizationResult"]
+      transform: >
+        $["x-ms-discriminator-value"] = "conversationalSummarizationResults";
+```
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["AnalyzeConversationConversationPIIResult"]
+      transform: >
+        $["x-ms-discriminator-value"] = "conversationalPIIResults";
+```
+
+### Sync API swagger mismatches
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["BasePrediction"]["properties"]["projectKind"]
+      transform: >
+        $["enum"] = ["Conversation", "Orchestration"];
+```
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["ConversationPrediction"]
+      transform: >
+        $["x-ms-discriminator-value"] = "Conversation";
+```
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["OrchestratorPrediction"]
+      transform: >
+        $["x-ms-discriminator-value"] = "Orchestration";
+```
+
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["AnalyzeConversationTaskKind"]
+      transform: >
+        $["enum"] = ["Conversation"];
+```
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["CustomConversationalTask"]
+      transform: >
+        $["x-ms-discriminator-value"] = "Conversation";
+```
+
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["AnalyzeConversationTaskResultsKind"]
+      transform: >
+        $["enum"] = ["ConversationResult"];
+```
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["CustomConversationalTaskResult"]
+      transform: >
+        $["x-ms-discriminator-value"] = "ConversationResult";
+```
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["AnalysisParameters"]["properties"]["targetProjectKind"]
+      transform: >
+        $["enum"] = [
+            "Luis",
+            "Conversation",
+            "QuestionAnswering",
+            "NonLinked"
+          ];
+```
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["definitions"]["ConversationTargetIntentResult"]
+      transform: >
+        $["x-ms-discriminator-value"] = "Conversation";
+```
+
+### Service Version
+```yaml
+directive:
+    - from: swagger-document
+      where: $["info"]
+      transform: >
+          $["version"] = "2022_05_15_Preview";
```

Comment:
the code generates the service version as "2022_05_15_preview" (p is small) which got me an error because it should be capital, that's why I modified it.

Reply:
I see. Thanks for clarifying. @ShivangiReja I opened https://github.com/Azure/autorest.csharp/issues/2248 to track.